tests/quantization/utils.py

```diff
@@ -1,12 +1,10 @@
-import torch
-
 from vllm.model_executor.layers.quantization import QUANTIZATION_METHODS
 from vllm.platforms import current_platform
 
 
 def is_quant_method_supported(quant_method: str) -> bool:
     # Currently, all quantization methods require Nvidia or AMD GPUs
-    if not torch.cuda.is_available():
```

Comment:
I really hate `torch.cuda.is_available()` . @DarkLight1337  if you have time,  please change all occurrence to avoid accidentally initializing cuda.

Reply:
I'll look into this in another PR.